src/events.js

```diff
@@ -18,72 +18,78 @@ function fireEvent(element, event) {
   })
 }
 
-const createEvent = {}
+function createEvent(
```

Comment:
No substantial changes here. This was necessary to make it easier to create regular `window.Event` events that still had the niceties that `createEvent` had to offer.